CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Fixed
 - Fixed PHP 8.4 deprecated call to `stream_context_set_option()` (#1545)
 - Fixed return type for `ZCOUNT` to be `int` (#1547)
+- fix(stream): Fixed exception type to match function signature (#1548)
```

Comment:
Can we be more specific? Something like:

```
Fixed throwing `CommunicationException` when stream is EOF.

Reply:
Done